server/ap_server/models.py

```diff
@@ -260,6 +260,16 @@ class Thread(BaseModel):
     )
 
 
+class ThreadState(BaseModel):
+    thread_id: UUID = Field(..., description="The ID of the thread.", title="Thread Id")
+    checkpoint_id: UUID = Field(
```

Comment:
gotcha so this now is alone and we don't have to think about separate checkpoint_ns, etc. right